collect_app/src/main/java/org/odk/collect/android/utilities/PlayServicesUtil.java

```diff
@@ -23,10 +23,10 @@ private PlayServicesUtil() {
     /**
      * Check that Google Play services APK is installed and up to date.
      *
-     * @return true if Google Play Services is available and up to
+     * @return true if Google Play Services are available and up to
      * date on this device; false otherwise.
      */
-    public static boolean isGooglePlayServicesAvailable(Context context) {
+    public static boolean areGooglePlayServicesAvailable(Context context) {
```

Comment:
I can see your rationale for renaming this. Since it's a wrapper on Google's `isGooglePlayServicesAvailable`, though, I would tend to leave it. Don't feel strongly.

Reply:
Fixed